connect/cli/plugins/locale/utils.py

```diff
@@ -0,0 +1,30 @@
+from typing import Any, Dict, List, Union
+
+import click
+from cmr import render
+
+
+def icon_for_autotranslate(resource: Dict[str, Any]) -> str:
+    return '\u2713' if resource['auto_translation'] else '\u2716'
+
+
+def stats_of_translations(resource: Dict[str, Any]) -> Union[str, int]:
+    count = resource["stats"]["translations"]
+    return count if count else '-'
+
+
+def row_format_resource(*fields: Any) -> str:
+    return ('| {} ' * len(fields)).format(*fields) + '|\n'  # noqa: P103
+
+
+def table_formater_resource(resource_str_list: List[str], count_of_resources: int, paging: int, page_size: int) -> None:
+    header = resource_str_list[:1]
+    if paging % page_size == 0:
+        start = 1 if paging == page_size else paging - page_size
+        if paging > page_size:
+            start += 1
+        click.echo(render(''.join(header + resource_str_list[start:])))
+    else:
+        start = count_of_resources - ((paging - 1) % page_size)
+        if resource_str_list[start:] and len(resource_str_list[1:]) == count_of_resources:
+            click.echo(render(''.join(header + resource_str_list[start:])))
```

Comment:
For consistency, probably should not use type annotations in just one file of there is not a compelling reason for it.

Reply:
Ok, I'll take them out in order to mantain consistency.